### DISCUSSION TITLE
Lookup Error

### DISCUSSION THREAD

**Discussion by Primal-id** [ZachChristensen28/pihole_dns_app#49] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1058873/210180418-8cc4246a-d39b-47f8-8daa-a5c9b0c271d7.png)

Where do I get the file to populate the lookup?

**Comment by ZachChristensen28** [comment]
Could you share which version of the [Pi-hole DNS Addon for Splunk](https://splunkbase.splunk.com/app/4505) you have installed? 


Docs: https://splunk-pihole-ta.ztsplunker.com

**Comment by Primal-id** [comment, marked as answer]
Splunk Enterprise
Version:
    9.0.3
    
Pi-hole DNS App for Splunk  2.2.2
Pihole DNS Add-on 1.3.2

 After a lot of troubleshooting I tracked down the issue to needing an app installed. The name of the app is "Splunk Add-on for Sysmon". Maybe this issue is specific to my setup. I'm not sure on why it occurred in regards to index=* sourcetype="pihole*"
 
 I used the following commands to find what apps used this info and looked at requirements for those apps
/opt/splunk/bin/splunk btool props list --debug | grep record_type
/opt/splunk/bin/splunk btool props list --debug | grep eventcode
